digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -66,6 +66,7 @@
 
     private static final String ECS_COMMAND_EXECUTOR_SYSTEM_CONFIG_PREFIX = "agent.command_executor.ecs.";
     private static final String DEFAULT_COMMAND_TASK_TTL = ECS_COMMAND_EXECUTOR_SYSTEM_CONFIG_PREFIX + "default_command_task_ttl";
+    private static final String ECS_TASK_FINISHED = "ECS_TASK_FINISHED";
```

Comment:
It's better to use a self-explaining name:

```suggestion
    private static final String ECS_END_OF_TASK_LOG_MARK = "ECS_TASK_FINISHED";
```

Reply:
I think the value (`"ECS_TASK_FINISHED"`) needs to be something more _unlike_ because log stream finishes unexpectedly if a script outputs this message.
How about using `"--RWNzQ29tbWFuZEV4ZWN1dG9y--"  // base64("EcsCommandExecutor")` for example? Idea is partially taken from https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html

Reply:
I changed key to `ECS_END_OF_TASK_LOG_MARK ` and value to `"--RWNzQ29tbWFuZEV4ZWN1dG9y--"` here.
https://github.com/treasure-data/digdag/pull/1316/commits/c57f438030abfbb3c95c6fd4b960f9e291f96561